apisix/ssl.lua

```diff
@@ -55,23 +56,32 @@ function _M.server_name()
 end
 
 
-local _aes_128_cbc_with_iv = false
+local _aes_128_cbc_with_iv_tbl = false
```

Comment:
Better to avoid using boolean value as the default value of a table?

Reply:
Done